RELEASE.md

```diff
@@ -0,0 +1,101 @@
+# Releases
+
+## Prerequisites - what you need to release WooCommerce Blocks
+- Install & set up [GitHub hub](https://hub.github.com) tools.
+- Configure a GitHub token for release scripts to use.
+  - https://github.com/settings/tokens
+  - Select the following permissions: 
+    - `admin:org`
+    - `public_repo`
+    - `read:user`
+  - Ensure your token is available to scripts, e.g. `export GH_API_TOKEN={YOUR-TOKEN}` or similar.
+- Get WPORG plugin svn commit access - ask a team member.
+
+_Outcome_: __You are equipped to ship a release!__
+
+## Release process
+### Lead-up to release
+#### Ensure release development is complete
+- Release milestone should exist (!).
+- Ensure all issues/PRs intended for this release are merged, closed and assigned to release milestone.
+- All PRs should have changelog entry, or `skip-changelog` tag.
+- Check with team to confirm any outstanding or in progress work.
+
+Note: changelog should be formatted like this in PR description. Note the preceding `> ` - this is required by changelog script.
+
+```md
+### Changelog
+
+> bug: Fix bug in Safari and other Webkit browsers that was causing the All Products block to show 0 results when resetting the sort value.
+```
+
+_Outcome_: __Team is aware of release and in agreement about what fixes & features are included.__
+
+#### Ensure release branch includes all relevant fixes
+- Make release branch if needed (major/minor).
+  - For fix releases, the branch should already exist.
+  - Otherwise create branch: `release/X.X`.
+- Update your local checkout to tip of the release branch.
+- Cherry pick all PRs into the release branch:
+  - If branch is already labelled `status: cherry-picked 🍒` then continue to next PR.
+  - Ideally, use GitHub Hub to cherry pick the PR - `hub am -3 {http://URL-TO-PR}`. 
+  - If there are serious conflicts or extensive differences between `master` and release branch, you may need to take more care:
+    - Manually cherry pick individual commits using git - `git cherry-pick {COMMIT-HASH}`.
+    - Or in some cases, manually craft a new PR with appropriate changes, targeting release branch.
```

Comment:
I love how you've broken down the potential ways a pr might be "cherry-picked" into the release branch. It's a good idea to highlight there may be times where it's not a clean merge.

Reply:
`hub am -3` is equivalence to `git am -3` which is equivalence to a graceful cherry-pick, you would never need to cherry-pick manually, since `am -3` will prompt you to fix conflict anytime there is one.

Reply:
Leaving this as is – with cherry picking and furious parallel development 🎆  on `master` there's always a risk that we'll need to take care with changes, and sometimes manually apply them. Cherry picking individual commits may still be a useful approach even if it's technically similar to cherry picking a multiple commits in a PR with `hub am -3`.

As we get more experience using this process we can finesse the wording of these steps so they are super clear and helpful.